codalab/apps/web/views.py

```diff
@@ -679,6 +679,16 @@ def get_context_data(self, **kwargs):
             context['is_owner'] = is_owner
             context['phase'] = phase
             context['groups'] = phase.scores()
+
+            for group in context['groups']:
```

Comment:
Could you add a ton of submissions and see if this causes a huge amount of lag? Or, at least make a note of this and when we push to testing, make sure a competition with lots of  submissions isn't lagging?